vendor/cookbooks/rails/templates/default/app_passenger_nginx.conf.erb

```diff
@@ -1,12 +1,17 @@
+<% server_root_public_path = "#{node['rails']['applications_root']}/#{@name}/current/public" %>
```

Comment:
Would it be possible to set this from the recipe instead of in the template? Somehow this feels a little hacky